build.sbt

```diff
@@ -1,3 +1,5 @@
 libraryDependencies += "org.scalatest" %% "scalatest" % "2.1.5" % "test"
 
-libraryDependencies += "org.apache.httpcomponents" % "httpclient" % "4.3.4"
\ No newline at end of file
+libraryDependencies += "org.apache.httpcomponents" % "httpclient" % "4.3.4"
+
+libraryDependencies += "com.typesafe.play" %% "play-json" % "2.4.0-M3"
```

Comment:
There are only milestone releases for Scala 2.10, see Maven Central: http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22com.typesafe.play%22%20AND%20a%3A%22play-json_2.10%22